deliver/lib/deliver/upload_metadata.rb

```diff
@@ -462,14 +451,18 @@ def retry_if_nil(message, tries: 5, wait_time: 10)
       end
     end
 
+    # Checking if the metadata to update includes App Info
+    def is_updating_app_info(options)
+      LOCALISED_APP_VALUES.keys.each do |key|
```

Comment:
It seems like "app info" consists of the following info:

https://github.com/fastlane/fastlane/blob/ab9fb3249adaddf6c05655891b4d280415dfc84e/spaceship/lib/spaceship/connect_api/models/app_info.rb#L7-L17

So I think this code should include `LOCALISED_APP_VALUES` (like you did) but also the `NON_LOCALISED_APP_VALUES`, which's everything related to 'categories'. Or am I missing something?

Reply:
@rogerluan you are right, though the result is used only to enable localizations. 
The `upload` function fetches again edit info later for the not localised app values and it also handles an empty answer[ without failing](https://github.com/fastlane/fastlane/blob/ab9fb3249adaddf6c05655891b4d280415dfc84e/deliver/lib/deliver/upload_metadata.rb#L225).

I think there's an opportunity to refactor some of this code to make it clearer and more coherent, but it's probably worth it to validate this solution before making bigger changes. 

I've updated the name to clarify that it only refers to localised values.